exercises/shared/.docs/help.md

```diff
@@ -0,0 +1,7 @@
+# Help
+
+To get help if you're having trouble, you can use one of the following resources:
+
+- [Powershell Dcoumentation](https://docs.microsoft.com/en-us/powershell/)
+
+<!-- TODO: Add more resources -->
```

Comment:
```suggestion
[Add more resources]: TODO
```

This (ab)uses a reference link syntax to allow us to use it as a _Markdown comment_.  See the [building markdown style guide][1] for more information.

[1]: https://github.com/exercism/docs/blob/main/building/markdown/markdown.md